telegram/forumtopic.py

```diff
@@ -0,0 +1,134 @@
+    considered equal, if their :attr:`message_thread_id`, :attr:`name` and :attr:`icon_color`
+    are equal.
+
+    .. versionadded:: 13.15
+
+    Args:
+        message_thread_id (:obj:`int`): Unique identifier of the forum topic
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`, optional): Unique identifier of the custom emoji shown
+            as the topic icon.
+
+    Attributes:
+        message_thread_id (:obj:`int`): Unique identifier of the forum topic
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`): Optional. Unique identifier of the custom emoji shown
+            as the topic icon.
+    """
+
+    __slots__ = ("message_thread_id", "name", "icon_color", "icon_custom_emoji_id")
+
+    def __init__(
+        self,
+        message_thread_id: int,
+        name: str,
+        icon_color: int,
+        icon_custom_emoji_id: str = None,
+        **_kwargs: Any,
+    ):
+        self.message_thread_id = message_thread_id
+        self.name = name
+        self.icon_color = icon_color
+        self.icon_custom_emoji_id = icon_custom_emoji_id
+
+        self._id_attrs = (self.message_thread_id, self.name, self.icon_color)
+
+
+class ForumTopicCreated(TelegramObject):
+    """
+    This object represents the content of a service message about a new forum topic created in
+    the chat.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`name` and :attr:`icon_color` are equal.
+
+    .. versionadded:: 13.15
+
+    Args:
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`, optional): Unique identifier of the custom emoji shown
+            as the topic icon.
+
+    Attributes:
+        name (:obj:`str`): Name of the topic
+        icon_color (:obj:`int`): Color of the topic icon in RGB format
+        icon_custom_emoji_id (:obj:`str`): Optional. Unique identifier of the custom emoji shown
+            as the topic icon.
+    """
+
+    __slots__ = ("name", "icon_color", "icon_custom_emoji_id")
+
+    def __init__(
+        self,
+        name: str,
+        icon_color: int,
+        icon_custom_emoji_id: str = None,
+        **_kwargs: Any,
+    ):
+        self.name = name
+        self.icon_color = icon_color
+        self.icon_custom_emoji_id = icon_custom_emoji_id
+
+        self._id_attrs = (self.name, self.icon_color)
+
+
+class ForumTopicClosed(TelegramObject):
+    """
+    This object represents a service message about a forum topic closed in the chat.
+    Currently holds no information.
+
+    .. versionadded:: 13.15
+    """
+
+    __slots__ = ()
+
+    def __init__(self, **_kwargs: Any):  # skipcq: PTC-W0049
+        pass
```

Comment:
do we need this? isn't the inherited `TelegramObject.__init__` enough?

Reply:
Apparently not, if I remove it, doing de_json fails with

```
>       return cls(bot=bot, **data)  # type: ignore[call-arg]
E       TypeError: ForumTopicClosed() takes no arguments
```